pycparser/c_parser.py

```diff
@@ -914,31 +915,39 @@ def p_struct_or_union_specifier_1(self, p):
             coord=self._token_coord(p, 2))
 
     def p_struct_or_union_specifier_2(self, p):
-        """ struct_or_union_specifier : struct_or_union brace_open struct_declaration_list brace_close
+        """ struct_or_union_specifier : struct_or_union brace_open struct_declaration_list_opt brace_close
         """
         klass = self._select_struct_union_class(p[1])
-        p[0] = klass(
-            name=None,
-            decls=p[3],
-            coord=self._token_coord(p, 2))
+        if len(p) == 4:
+            # None means no members
```

Comment:
Seems like the code has changed, there's no None assignment now - is this comment still relevant?

Reply:
Still relevant, but not in this function.
Comment moved to `p_struct_or_union_specifier_1` (which handles the `None` case)